couchbase/admin/search/v1/search.proto

```diff
@@ -26,12 +50,19 @@ message UpsertIndexRequest {
   optional string source_type = 7;
   optional string source_uuid = 8;
   string type = 9;
-  optional string uuid = 10;
-  optional string bucket_name = 11;
-  optional string scope_name = 12;
+  optional string bucket_name = 10;
+  optional string scope_name = 11;
```

Comment:
Not sure why bucket_name and scope_name aren't in Index, since they're common to both Create and Update?

Reply:
Neither are in the result for get/list so I omitted them from the index itself.

Reply:
It's also worth considering that bucket/scope define the scope of the index (distinction scoped indexes or global indexex), which are independant of the place that the index sources its data (which is the part thats actually 'embedded' in the index).

Reply:
Well, there's plenty of other metadata in the json that's not related to the where the data is sourced from, like the name, type and uuid fields.  IMO the json should contain everything related to the index and I've no idea why the FTS team chose to omit these two fields.
However they did, so I'm fine with Charlie's argument.